src/main/resources/locales/en/translation/self-assignment-form.json

```diff
@@ -22,7 +22,7 @@
       "required": "Enter claimant last name that exactly matches the case details"
     },
     "hiddenErrorField": {
-      "api": "Unable to assign role. Please try again later",
+      "api": "Unable to assign role. An unexpected error occurred. Please try again later",
```

Comment:
Do we need to update the same wordings in self-assignment-check.json and related Welsh translations?

Reply:
Removed extra information to match welsh translation.